Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Alpine for smaller size
+FROM python:3.9-alpine
+
+# Create a system account hubibot.hubibot
+RUN addgroup -S hubibot && adduser -S hubibot -G hubibot
+# Non-alpine equivalent of above:
+#RUN groupadd -r hubibot && useradd -r -m -g hubibot hubibot
+
+USER hubibot
+
+WORKDIR /app
+
+COPY requirements.txt     /app
+COPY main.py              /app
+COPY template.config.yaml /app
```

Comment:
Not sure what's the point of copying template.config.yaml.  As I understand the real config needs to be mounted, otherwise it won't work.  template makes some kind of sense if it use used as a source for copying into the real config file, but in the context of Docker it does not make sense - the change will be lost when container stops.

Reply:
it will make sense next PR. Stay tuned